io.openems.edge.bridge.http/src/io/openems/edge/bridge/http/api/BridgeHttp.java

```diff
@@ -1,28 +1,37 @@
 package io.openems.edge.bridge.http.api;
 
+import static java.util.Collections.emptyMap;
```

Comment:
please dont

> So when should you use static import? Very sparingly! Only use it when you'd otherwise be tempted to declare local copies of constants, or to abuse inheritance (the Constant Interface Antipattern).

https://docs.oracle.com/javase/8/docs/technotes/guides/language/static-import.html

Reply:
what about usidn Collections.emptyMap() with import java.util.Collections; ?

Reply:
I understand, but i would also consider `emptyMap` as well as `emptyList` are still readable enough without the class name, because the method name says everything of what to expect. A static import of `of` i would understand it could be from `Map.of` or `List.of` and it would affect the readability.
And also especially these two methods should be known (imo) because they decrease the need of null checks and unexpected NullPointerExceptions.
(Its also more to write :) )

But anyway if you still think it enhances readability i will add it :)